Youtube/README.md

```diff
@@ -0,0 +1,631 @@
+- 401 Unauthorized: Token de acesso inválido ou expirado
+
+**Casos de uso**:
+- Remover vídeos privados ou não listados que não são mais necessários
+- Gerenciar espaço na conta do YouTube 
+- Remover vídeos que violam as diretrizes da plataforma
+- Manter a organização do canal removendo conteúdo desatualizado
+
+### Trabalhando com Legendas (Captions)
+
+A API de legendas do YouTube funciona em duas etapas:
+1. Primeiro, liste as legendas disponíveis para um vídeo
+2. Em seguida, faça uma requisição específica para obter o texto completo da legenda
+
+**Observações importantes**:
+- O download de legendas requer autenticação, mesmo para vídeos públicos
+- Diferentes formatos estão disponíveis: SRT (mais comum), SBV (formato YouTube) e VTT (para HTML5)
+- As legendas podem ser em vários idiomas, verifique o campo `language` na resposta da listagem
+
+**Exemplo completo**:
+```typescript
+// Listar legendas disponíveis
+const legendasDisponiveis = await loader({
+  videoId: "ID_DO_VIDEO"
+}, req);
+
+// Verificar se há legendas
+if (!legendasDisponiveis || !legendasDisponiveis.items || legendasDisponiveis.items.length === 0) {
+  console.log("Este vídeo não possui legendas");
+  return;
+}
+
+// Exibir idiomas disponíveis
+console.log("Legendas disponíveis:");
+for (const legenda of legendasDisponiveis.items) {
+  console.log(`- ${legenda.snippet.language} (${legenda.snippet.name}) - ID: ${legenda.id}`);
+}
+
+// Obter texto da legenda em português (se disponível)
+const legendaPt = legendasDisponiveis.items.find(item => 
+  item.snippet.language === "pt" || item.snippet.language === "pt-BR"
+);
+
+if (legendaPt) {
+  // Obter texto completo da legenda
+  const textoLegenda = await loader({
+    captionId: legendaPt.id,
+    format: "srt" // formatos disponíveis: "srt", "sbv", "vtt"
+  }, req);
+  
+  console.log("Texto da legenda:", textoLegenda);
+  
+  // Para processar o formato SRT, você precisará de um parser
+  // ou pode exibir o texto bruto conforme retornado pela API
+}
+```
+
+- Operações como curtir comentários, definir status de moderação e outras ações sem conteúdo devem usar a abordagem de parâmetros na URL, não no corpo da requisição.
+
+## Observações
+
+- Para modificar informações do canal, você deve ser o proprietário do canal ou ter permissões adequadas.
+- Para pinar comentários, você deve ser o proprietário do vídeo ou ter permissões adequadas.
+- Para acessar dados de analytics, você precisa ter o escopo `yt-analytics.readonly` autorizado.
+- Algumas operações exigem tokens de acesso com escopos específicos.
+- A API do YouTube tem restrições específicas sobre quais partes podem ser atualizadas em conjunto. Por exemplo, `snippet` e `brandingSettings` devem ser atualizados em chamadas separadas.
+- Operações como curtir comentários, definir status de moderação e outras ações sem conteúdo devem usar a abordagem de parâmetros na URL, não no corpo da requisição.
```

Comment:
Put that above the resources.

It seems that most problems that uses may have are relate to this... so, keep it up on the doc.